### ISSUE TITLE
`npm install` fails

### ISSUE THREAD

**Issue by conatus** [FormidableLabs/spectacle-boilerplate#22]
Hey guys, wonderful looking library - really excited to get it running.

Sadly though `npm install` seems to fail here. The log is below

```
> radium@0.17.1 postinstall <path snipped>/node_modules/radium
> cd lib || npm run lib

sh: line 0: cd: lib: No such file or directory

> radium@0.17.1 lib <path snipped>/node_modules/radium
> npm run babel && rimraf lib/__tests__ lib/__mocks__


> radium@0.17.1 babel <path snipped>/node_modules/radium
> rimraf lib && babel src/ -d lib/

sh: babel: command not found

npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/Cellar/nvm/0.23.3/versions/node/v5.10.1/bin/node" "/usr/local/Cellar/nvm/0.23.3/versions/node/v5.10.1/bin/npm" "run" "babel"
npm ERR! node v5.10.1
npm ERR! npm  v3.8.3
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! radium@0.17.1 babel: `rimraf lib && babel src/ -d lib/`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the radium@0.17.1 babel script 'rimraf lib && babel src/ -d lib/'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the radium package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rimraf lib && babel src/ -d lib/
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs radium
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls radium
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     <path snipped>/node_modules/radium/npm-debug.log

npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/Cellar/nvm/0.23.3/versions/node/v5.10.1/bin/node" "/usr/local/Cellar/nvm/0.23.3/versions/node/v5.10.1/bin/npm" "run" "lib"
npm ERR! node v5.10.1
npm ERR! npm  v3.8.3
npm ERR! code ELIFECYCLE
npm ERR! radium@0.17.1 lib: `npm run babel && rimraf lib/__tests__ lib/__mocks__`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the radium@0.17.1 lib script 'npm run babel && rimraf lib/__tests__ lib/__mocks__'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the radium package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run babel && rimraf lib/__tests__ lib/__mocks__
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs radium
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls radium
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     <path snipped>/node_modules/radium/npm-debug.log
npm WARN spectacle-boilerplate@1.0.1 No repository field.
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/Cellar/nvm/0.23.3/versions/node/v5.10.1/bin/node" "/usr/local/opt/nvm/versions/node/v5.10.1/bin/npm" "install"
npm ERR! node v5.10.1
npm ERR! npm  v3.8.3
npm ERR! code ELIFECYCLE

npm ERR! radium@0.17.1 postinstall: `cd lib || npm run lib`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the radium@0.17.1 postinstall script 'cd lib || npm run lib'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the radium package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cd lib || npm run lib
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs radium
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls radium
npm ERR! There is likely additional logging output above.
```

Radium appears to have been removed from `node_modules` which may simply be part of the general npm lifecycle.

The same occurs with a plain install of Spectacle.

Using the latest Node (5.10.1) and npm (3.8.7).

Thanks a lot.

**Comment by conatus** [comment]
The temporary fix is to hard code the dependency as the last release (`1.0.4`) which is the last working version. `^1.0` in the dependencies resolves to `1.0.5` the latest tag which appears to be where the issue began. I suspect that the bumping of Radium done in https://github.com/FormidableLabs/spectacle/commit/269aaef29695ad9dfa72b125834830f02064713b introduced the regression somehow.

Thanks for making Spectacle though, looking forward to getting involved!

**Comment by kenwheeler** [comment]
odd, this should work. what version of npm are you on? do you have babel installed globally?

**Comment by kenwheeler** [comment]
This is fixed now in 1.0.7
